docs/developer-docs/daos/sns/managing/making-proposals.mdx

```diff
@@ -326,6 +326,81 @@ The SNS DAO has control over a treasury from which funds can be sent to other ac
 
 This is a ["critical" proposal type](#criticalproposals), which means it is subject to higher voting thresholds before it is accepted.
 
+#### Maximum 7-day Amount Total
+
+The rate that funds can be transferred from the treasury is limited. To explain how this works,
+first, define T to be the value of the tokens in the treasury in units of [XDR]. T can fall within
+three size regimes:
+
+[XDR]: https://en.wikipedia.org/wiki/Special_drawing_rights
+
+<table>
+  <thead>
+    <tr>
+      <th>Size</th>
+      <th>Max T (XDR)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>small</td>
+      <td>100_000</td>
```

Comment:
how about giving the range here (0-100_000) rather than the max? I think the max is a bit harder to parse... We mean any T that falls into this bucket, right?